cmake/Modules/FindMKLDNN.cmake

```diff
@@ -42,18 +42,14 @@ IF(NOT MKLDNN_FOUND)
         list(APPEND DNNL_MAKE_COMMAND "--" "-l" "$ENV{MAX_JOBS}")
       endif()
     endif()
-    if(LINUX)
-      set(DNNL_CXX_FLAGS "-DCMAKE_CXX_FLAGS=-fpreview-breaking-changes")
-    else()
-      set(DNNL_CXX_FLAGS "")
-    endif()
+    set(DNNL_CXX_FLAGS ${CMAKE_CXX_FLAGS} ${XPU_DEVICE_CXX_FLAGS})
```

Comment:
I think the semantics do not exactly match the original one. However, I recalled that Minjean has verified a similar change. @min-jean-cho , any comments?

Reply:
The `CMAKE_CXX_FLAGS` here is not the default `CMAKE_CXX_FLAGS` by CMake, but includes custom flags added by PyTorch CMake configuration. Passing the modified `CMAKE_CXX_FLAGS` to DNNL build leads to unexpected behavior?